src/integrationTest/java/org/opensearch/security/FlsAndFieldMaskingTests.java

```diff
@@ -206,6 +209,36 @@ public class FlsAndFieldMaskingTests {
             .on("*")
     );
 
+    /**
+     * Example user with fls filter in which the user can only see the {@link Song#FIELD_TITLE} field.
+     */
+    static final TestSecurityConfig.User INCLUSIVE_FLS_USER = new TestSecurityConfig.User("inclusive_fls_user").roles(
+        new TestSecurityConfig.Role("example_inclusive_fls").clusterPermissions("cluster_composite_ops_ro")
+            .indexPermissions("read")
+            .fls(FIELD_TITLE)
+            .on(FIRST_INDEX_NAME)
+    );
+
+    /**
+     * Example user with fls filter in which the user can see every field but the {@link Song#FIELD_TITLE} field.
+     */
+    static final TestSecurityConfig.User EXCLUSIVE_FLS_USER = new TestSecurityConfig.User("exclusive_fls_user").roles(
```

Comment:
Nit: Similar to the other PR maybe we can swap the names so they illustrate what the roles/users do? For example USER_NO_FIELD_Title or something similar.